<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="webTab">
       <attribute name="title">
        <string>Web</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="addressBar"/>
          </item>
          <item>
           <widget class="QCommandLinkButton" name="goBtn">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="webVertical"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="crawlTab">
       <attribute name="title">
        <string>Crawl</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="crawlVertical"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCrawl">
    <property name="title">
     <string>Crawl</string>
    </property>
    <widget class="QMenu" name="menuStart_crawler">
     <property name="title">
      <string>Start crawler</string>
     </property>
     <addaction name="onThisPage"/>
     <addaction name="actionEnter_Url"/>
    </widget>
    <addaction name="menuStart_crawler"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSave_Settings"/>
    <addaction name="actionReload_Settings"/>
   </widget>
   <addaction name="menuCrawl"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="onThisPage">
   <property name="text">
    <string>From Account File</string>
   </property>
  </action>
  <action name="actionEnter_Url">
   <property name="text">
    <string>EnterUrl</string>
   </property>
  </action>
  <action name="actionSave_Settings">
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionProxy">
   <property name="text">
    <string>Proxy</string>
   </property>
  </action>
  <action name="actionReload_Settings">
   <property name="text">
    <string>Reload Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
